tasks/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
+- setup: 
```

Comment:
This step mean you need to prevent to gathering facts on hosts that are not used, isn't it?

Reply:
Actually this is for tasks/reboot.yml as I add line 46 in this commit. Reboot module does not work with lxc so I need to skip it when using lxc. But the way to detect it is in ansible facts so I need to gathering facts here.

Reply:
Gotcha because [this line](https://github.com/opsta/opsta-playbook/blob/2dfe3952e5f8da7e15e05b42d4b9ae5db6f14df3/host-preparation.yml#L3) cannot be "true" due to the remote host wasn't installed python yet.